src/main/java/pl/techlab24/OSKManager/model/validation/ClientValidator.java

```diff
@@ -116,27 +119,27 @@ private static String validateDocumentNumber(String documentNumber) {
 
     private static boolean checkSum(String pesel) {
         int lastDigit;
-        int[] peselNumbers =  new int[11];
+        int[] peselNumbers = new int[11];
 
         for (int i = 0; i < pesel.length(); i++) {
             peselNumbers[i] = Integer.parseInt(String.valueOf(pesel.charAt(i)));
         }
 
-        int sum = peselNumbers[0] +
-            peselNumbers[1] * 3 +
-            peselNumbers[2] * 7 +
-            peselNumbers[3] * 9 +
-            peselNumbers[4] +
-            peselNumbers[5] * 3 +
-            peselNumbers[6] * 7 +
-            peselNumbers[7] * 9 +
-            peselNumbers[8] +
-            peselNumbers[9] * 3;
+        int sum = peselNumbers[0]
+            + peselNumbers[1] * 3
+            + peselNumbers[2] * 7
+            + peselNumbers[3] * 9
+            + peselNumbers[4]
+            + peselNumbers[5] * 3
+            + peselNumbers[6] * 7
+            + peselNumbers[7] * 9
+            + peselNumbers[8]
+            + peselNumbers[9] * 3;
 
-        if ((sum %=10) == 0) {
+        if ((sum %= 10) == 0) {
             lastDigit = 0;
         } else {
-            lastDigit = 10 - (sum%10);
+            lastDigit = 10 - (sum % 10);
```

Comment:
To start our discussion on this PR I would like to notice that % 10 is redundant in the above line (line no 142). This line should simply look like this: _lastDigit = 10 - sum_. Why? In line no 139 we are defining sum as the reminder (because of = sign), so (starting from line 139) sum is the integer number that is greater or equal to 0 (>= 0) and less or equal to 9 (<= 9).

Reply:
```suggestion
            lastDigit = 10 - sum;
```

Reply:
Due to our plans to remove checkSum method, I am resolving this remark.